src/distributions/float.rs

```diff
@@ -69,7 +69,8 @@ pub struct OpenClosed01;
 pub struct Open01;
 
 
-pub(crate) trait IntoFloat {
+#[doc(hidden)]
+pub trait IntoFloat {
```

Comment:
Why was this change necessary?

Reply:
Because it's used in three places (two still in Rand, one in the moved code).

Reply:
Maybe add a comment that it is still needed by `rand_distr`?